homeassistant/components/overkiz/water_heater_entities/domestic_hot_water_production.py

```diff
@@ -29,8 +29,6 @@
     OverkizCommandParam.BOOST: STATE_PERFORMANCE,
 }
 
-OPERATION_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_OPERATION_MODE.items()}
```

Comment:
You can keep this, even if there is values you don’t want. So we still have similar code with other climate entisies.
But can keep your logic around `_attr_operation_list`.

Reply:
it's never used, it will add confusion against the other custom variables.
I'll have a mypy alert saying this var is not used I think.

Reply:
I would prefer having "not used everywhere const" into a const file.

I think that's not too bad if this reversed dict isn't here.

Reply:
The constant is not used now, but if you mimic the other climate entities it should be.

Reply:
It could not work like other devices.  

If the regular reverse const is used in `async_set_operation_mode` this means that all DHWP who want to use the preset to `STATE_ECO` will set the `OverkizCommandParam.AUTO_MODE` to overkiz.  
But... some devices don't support this specific mode, and are requiring `OverkizCommandParam.AUTO`.

So this constant won't be used anywhere, and will be a source of bug IMHO.